Functions/Gherkin.Output.ps1

```diff
@@ -0,0 +1,705 @@
+                    # }
+                    # else {
+                    #     $null
+                    # }
+                    # & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Pending "$margin[?] $output" -NoNewLine
+                    # & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Pending ", is pending$because" -NoNewLine
+                    # & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.PendingTime " $humanTime"
+                    break
+                }
+
+                Inconclusive {
+                    $FgText= $Script:ReportTheme.Undefined
+                    $FgTime = $Script:ReportTheme.UndefinedTime
+
+                    $WriteStepParams += @{
+                        StepTextColor = $FgText
+                        StepDurationColor = $FgTime
+                    }
+
+                    $StepResult |
+                        Write-GherkinStepText  @WriteStepParams |
+                        Write-GherkinStepUndefined -IndentationLevel $BaseIndent
+
+                    # $because = if ($StepResult.ErrorRecord.TargetObject.Data.Because) {
+                    #     ", because $($StepResult.ErrorRecord.TargetObject.Data.Because)"
+                    # }
+                    # else {
+                    #     $null
+                    # }
+                    # & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Inconclusive "$margin[?] $output" -NoNewLine
+                    # & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Inconclusive ", is inconclusive$because" -NoNewLine
+                    # & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.InconclusiveTime " $humanTime"
+
+                    break
+                }
+
+                default {
+                    # TODO:  Add actual Incomplete status as default rather than checking for null time.
+                    if ($null -eq $StepResult.Time) {
+                        & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.Incomplete "$margin[?] $output" -NoNewLine
+                        & $SafeCommands['Write-Host'] -ForegroundColor $ReportTheme.IncompleteTime " $humanTime"
+                    }
+                }
+            }
+        }
+    }
+}
+
+function Get-GherkinStepTextParts {
+    [CmdletBinding()]
+    [OutputType([hashtable[]])]
+    Param (
+        [Parameter(Position = 0, Mandatory = $True, ValueFromPipeline = $True)]
+        $StepResult
+    )
+
+    Process {
+        $StepText = $StepResult.TypeNames
+        $StepParameters = $StepResult.Parameters
+        $Result = $StepResult.Result
+
+        # Split the step text into pieces, where the pieces are either generic step text, or input to the step
+        # definition. This will allow "highlighting" of the replaceable text parameters of the step
+        # definition.
+        $StepTextParts = [hashtable[]]@()
+        if ($Result -notin 'Inconclusive' -and $StepParameters.Keys.Count) {
```

Comment:
@fourpastmidnight  The `-notin` operator is only available in PS 3+. That is why the PS 2 job is failing to start

Reply:
@fourpastmidnight Is $Result an array in this context? 

If so, you should switch to do `$Result -notcontains 'Inconclusive'`. Otherwise, if it is a single value, just do a comparison with `-ne`

Reply:
Oooh, good catch on the usage of `-notin`. Also, I had previously implemented these changes on a privately maintained copy of Pester (to initially just get it to work!--and this implementation is much better than my original). Anyway, I was going to try to replace usages of `Inconclusive` with `Undefined` since that is what Cucumber uses. BUT, the NUnit Xml reports I think require `Inconclusive` as a value for the test result, so I'm going to stick with a single value and I need to remove that `-notcontains` and make it `-ne` as you suggest. Again, thanks for the second set of eyes!

Reply:
Thanks again for pointing those issues out. They weren't the main thing causing the tests to break, but most certainly would not have helped my cause with PSv2. This is fixed in the latest version of this commit.